Jenkinsfile

```diff
@@ -387,19 +387,14 @@ pipeline {
           stages {
             stage('Libbeat oss'){
               steps {
-                makeTarget(context: "Libbeat oss Linux", target: "-C libbeat testsuite")
+                mageTarget(context: "Libbeat oss Linux", directory: "libbeat", target: "build test")
               }
             }
             stage('Libbeat crosscompile'){
               steps {
                 makeTarget(context: "Libbeat oss crosscompile", target: "-C libbeat crosscompile")
               }
             }
-            stage('Libbeat stress-tests'){
```

Comment:
Could you comment more on this change, please? Is this being moved elsewhere and I'm just not seeing it or is it being removed entirely?

Reply:
Good question. I forgot about this change. I should have mentioned it in the commit notes or reverted it. I found that this target never does anything, but I need to look into this more to figure out what it is supposed to be doing and fix it.

I'll revert this section for now.